setup.py

```diff
@@ -7,7 +7,7 @@
 setup(name='stdeb',
       # Keep version in sync with stdeb/__init__.py and install section
       # of README.rst.
-      version='0.8.5',
+      version='0.8.5.1',
```

Comment:
This change seems unrelated and should be reverted.

Reply:
Done